modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml

```diff
@@ -9,9 +9,11 @@ dependencies:
     - field.field.node.alert.field_alert_link
     - field.field.node.alert.field_alert_place
     - field.field.node.alert.field_alert_text_color
+    - field.field.node.alert.field_meta_tags
```

Comment:
I'm not sure that this CT needs meta tags. There is no individual page for it. It's available only as block on the page in header or footer.